controllers/schedule/active/controller.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2021 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package active
+
+import (
+	"context"
+	"github.com/chaos-mesh/chaos-mesh/api/v1alpha1"
+	"github.com/chaos-mesh/chaos-mesh/controllers/types"
+	"github.com/go-logr/logr"
+	"go.uber.org/fx"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/client-go/tools/reference"
+	"k8s.io/client-go/util/retry"
+	"reflect"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type Reconciler struct {
+	scheme *runtime.Scheme
+
+	client.Client
+	Log logr.Logger
+
+	Recorder record.EventRecorder
+}
+
+func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
+	ctx := context.Background()
+
+	schedule := &v1alpha1.Schedule{}
+	err := r.Get(ctx, req.NamespacedName, schedule)
+	if err != nil {
+		r.Log.Error(err, "unable to get chaos")
+		return ctrl.Result{}, nil
+	}
+
+	kind, ok := v1alpha1.AllKinds()[string(schedule.Spec.Type)]
```

Comment:
`v1alpha1.AllKinds()` seems to contains only chaos type only, workflow type is not supported yet.

Reply:
As the support for `Workflow`, I need another discussion with @STRRL (as it needs the finished status to GC the history). I prefer to include it in another PR as this one is too big.